docs/user-guide/rules.md

```diff
@@ -388,196 +388,4 @@ Require or disallow whitespace on the inside with this `whitespace-inside` rule.
 | | | |
 | :-- | :-: | :-: |
 | [`comment-whitespace-inside`](../../lib/rules/comment-whitespace-inside/README.md)<br/>Require or disallow whitespace on the inside of comment markers. | ✅ | 🔧 |
-
-## Deprecated
-
-These rules are deprecated — we won't fix bugs nor add options, and we will remove them in the next major release. We recommend you use a pretty printer (like Prettier) alongside Stylelint rather than these rules.
```

Comment:
Removing this header triggers the lint error in CI (since links to `#deprecated` will now be broken). Do we want to keep this header here? If so, any suggestions on what to include?

Reply:
I agree with removing the `## Deprecated` header. But it seems good that `to-15.md` will include the removed rules list, instead of the links to `rules.md`.

https://github.com/stylelint/stylelint/blob/5be9a7b996d7846c06fdec00208b27b3c7544727/docs/migration-guide/to-15.md?plain=1#L14

Reply:
Gotcha - just to confirm, should I then create a removed rules list in the `to-15` migration guide?

Reply:
Yes. If so, I believe people could easily find the deprecated rules with v15 later.

Reply:
Thanks for the suggestion; I gave a first-draft approach in https://github.com/stylelint/stylelint/pull/6979/commits/9074d968a699aee4170629bf7e4791543b766da4; let me know what you think!